src/components/Nav/NavUtils.tsx

```diff
@@ -11,11 +11,12 @@ const buildCommonQueryParams = (params: GraphParamsType): string => {
 };
 
 export const makeNamespaceGraphUrlFromParams = (params: GraphParamsType): string => {
-  const namespace = store.getState().namespaces.activeNamespace.name;
+  const namespaces = store
+    .getState()
+    .namespaces.activeNamespaces.map(namespace => namespace.name)
+    .join(',');
```

Comment:
ooh, nice way of making CSV :-)

Reply:
thanks, probably going to put a separate function to handle that without having to repeat it all over the place.